### ISSUE TITLE
Metrics - invalid queues

### ISSUE THREAD

**Issue by mnabialek** [laravel/horizon#217]
When I go into metrics -> queue, it seems there are 4 queues but there names are displayed as:

```
r
ee
(blank)
(blank)
```

(In place of blank there is empty line).

However in my environments I have:

```
        'local' => [
            'supervisor-1' => [
                'connection' => 'redis',
                'queue' => ['one','two','three','four'],
                'balance' => 'auto',
                'processes' => 10,
                'tries' => 1,
            ],
        ],
```

I've already run `redis-cli flushall`  but it didn't help.

For sure I was playing before with horizon, so maybe those queues names represent something I was testing, but now I have one, two, three four and I cannot force Horizon to display them here

**Comment by mnabialek** [comment]
It seems the problem is here:

```
    public function measuredQueues()
    {
        $queues = (array) $this->connection()->smembers('measured_queues');
dd($queues);
        return collect($queues)->map(function ($class) {
            return substr($class, 14);
        })->all();
    }
```

I've added `dd` here, and I see the array contains:

````
bubu:queue:four
bubu:queue:three
bubu:queue:two
bubu:queue:one
```

(I've set prefix just for some other tests to `bubu` in horizon config).

So it seems that it's assumed that prefix won't be changed and if it's changed invalid queue names will be taken (`start` for `substr` has invalid value here)

**Comment by mnabialek** [comment]
Exact same problem seems to be  for Metrics - jobs. When you go into jobs I don't see any metrics (although I've run `horizon:snaphsot`) but when I've changed prefix in config to `horizon:` it's working fine again

**Comment by pdbreen** [comment]
This is fixed by #197 - just hasn't been tagged yet.

**Comment by themsaid** [comment]
Tagging a new release today or tomorrow. Thanks everyone :)
